core/dbt/adapters/base/impl.py

```diff
@@ -453,9 +460,10 @@ def _relations_cache_for_schemas(
         # it's possible that there were no relations in some schemas. We want
         # to insert the schemas we query into the cache's `.schemas` attribute
         # so we can check it later
-        cache_update: Set[Tuple[Optional[str], Optional[str]]] = set()
+        cache_update: Set[Tuple[Optional[str], str]] = set()
```

Comment:
This type aligns with the change below and with what update_schemas expects (which is the only reason for this variable).